solutions/event-routing-account-settings/catalogValidationValues.json.template

```diff
@@ -0,0 +1,4 @@
+{
+  "ibmcloud_api_key": $VALIDATION_APIKEY",
+  "metadata_region_primary": "eu-de"
```

Comment:
Since this is an optional input, I would not set it here to avoid any test clashes within our account